src/StockportWebapp/Views/stockroom/Events/Detail2024.cshtml

```diff
@@ -30,21 +30,24 @@
                 </div>
 
                 <div class="event-hero-container">
-                    <div class="article-hero__content event-hero__content--70">
-                        <div class="article-hero__about event-hero__about--extra-padding">
+                    <div class="hero__content event-hero__content--70">
+                        <div class="hero__about event-hero__about--extra-padding">
                             <p>
                                 @Model.EventDate.ToString("dddd dd MMMM yyyy") -
                                 From @DateTime.Parse(Model.StartTime).ToString("h:mmtt").ToLower()
                             </p>
 
-                           <a href="@Model.BookingInformation" target="_blank" class="btn event-btn" aria-label="Book @Model.Title">Book now</a>
+                            @if (!string.IsNullOrEmpty(Model.BookingInformation))
+                            {
+                                <a href="@Model.BookingInformation" target="_blank" class="btn event-btn" aria-label="Book @Model.Title">Book now</a>
+                            }
 
                             @Html.Raw(Model.Description)
                         </div>
                     </div>
 
-                    <div class="article-hero__content event-hero__content--30">
-                        <div class="article-hero__about event-hero__about">
+                    <div class="hero__content event-hero__content--30">
+                        <div class="hero__about event-hero__about">
```

Comment:
Just checking the event-hero__content--30 and event-hero__about haven't also moved module